docs/source/observability.md

```diff
@@ -6,8 +6,8 @@ These metrics include the number of requests per second, the number of errors, t
 
 ## Dashboard
 
-You can use the Kong official `statsd` dashboard in your cockpit.
+The deployment tool will automatically import a Grafana dashboard for Kong. This dashboard gives you a quick overview of the health of your Kong gateway.
```

Comment:
It would be nice to keep the link to the grafana dashboard from the old version. We could just hyperlink "dashboard for Kong".

Link: https://grafana.com/grafana/dashboards/16897-kong-statsd-exporter/

Reply:
Great idea, added back the link.